human.txt

```diff
@@ -31,8 +32,8 @@ jsstrn
 justyn-clark
 kenhrmt
 kylecho
-Leo-Mu
 leo424y
+Leo-Mu
```

Comment:
Actually, he sorted the entire list. So that should be fine since we get a fully sorted list in the end.